server/game/cards/06-WoE/Absolve.js

```diff
@@ -0,0 +1,50 @@
+const Card = require('../../Card.js');
+
+class Absolve extends Card {
+    // Play: Heal 1D from each creature. Each creature healed this way captures 1A from its opponent
+    setupCardAbilities(ability) {
+        this.play({
+            //effect: 'heal 1 damage from all creatures',
+            gameAction: ability.actions.heal((context) => ({
+                amount: 1,
+                target: context.game.creaturesInPlay.filter((card) => card.hasToken('damage'))
+            })),
+            then: {
+                message: '{1} heals {3}, capturing {0} {4} amber',
```

Comment:
I think this message can be avoided because the previous effect will state which creatures will be healed.